bin/beaker

```diff
@@ -3,7 +3,11 @@
 require 'rubygems' unless defined?(Gem)
 require 'beaker'
 
-Beaker::CLI.new.execute!
+if (Beaker::Subcommands.instance_methods(false) << :help).include? ARGV[0].to_sym
```

Comment:
initially, this line looked like:

if ["help", "init"].include? ARGV[0]

which is a lot easier to read, but the current implementation means that we don't need to update this line every time we add a new subcommand